### PR TITLE
Review this to get started

### PR DESCRIPTION
**Pull request by robertosilvino**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/robertosilvino/ReviewNinja-Welcome/pull/1" target="_blank">ReviewNinja

File: ReadMe.md [hunk 1]
```diff
@@ -1 +1,41 @@
-# ReviewNinja-Welcome
\ No newline at end of file
```
Comment (robertosilvino) [inline ReadMe.md:1]: test

Comment (robertosilvino) [inline ReadMe.md:1]: !fix very good!

Comment (robertosilvino) [inline ReadMe.md:1]: !fixed  (Y)

**Pull request merged**